src/consumer.cpp

```diff
@@ -48,6 +52,23 @@ void Consumer::rebalance_proxy(rd_kafka_t*, rd_kafka_resp_err_t error,
     static_cast<Consumer*>(opaque)->handle_rebalance(error, list);
 }
 
+TopicPartitionList Consumer::get_matching_partitions(TopicPartitionList&& partitions,
```

Comment:
Why is this an rvalue reference and not a const ref?

Reply:
I move it internally. I don't need to make a copy.

Reply:
I didn't see the move. But anyway, in that case it should be taken by value, not by rvalue ref. The value will be moved into this function so won't be making any copies.